test-packages/e2e-tests/server.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const express = require('express');
+const { createODataApp } = require('./odata');
+const { createRestApp } = require('./rest');
+
+async function startServer() {
+  const app = express();
+  const odataApp = createODataApp();
```

Comment:
why are you putting both APIs (OData and rest) in the same application?

Reply:
Because the setup is much simpler with this than starting an extra server - maintenance and performance-wise. Other than that the apps do not interfere with one another.
Would you prefer to separate them?

Reply:
ok